### PR TITLE
Update Changelog

### PR DESCRIPTION
**Pull request by jaapmarcus**

### CODE REVIEW

**Review by John-S4 — APPROVED** [review]
Should API be capitalised?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
+## Version 0.2.6 (12-28-2022)
+
+### Security
+ - Introduce API wrapper for custom themes and modules to facilitate easier CSRF tokens (#612)
+
+## Bugfixes
+ - Fixed an issue with alias under Apache2 (#626)
+ - Add an additional check for CSFR when php://input is empty (#626)
+ - Fixed an issue where you where not able to reuse existing packages as root user (#607)
```
Comment (BelleNottelling) [inline CHANGELOG.md:9]: Please update this to specify it's a fix for the WHM server manager

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+## Bugfixes
+ - Fixed an issue with alias under Apache2 (#626)
+ - Add an additional check for CSFR when php://input is empty (#626)
+ - Fixed an issue where you where not able to reuse existing packages as root user (#607) 
+ - Refactor loading locales (#623)
+ - Update how we load available locale selection (#611)
+ 
+### Other
+ - Updated some dependencies.
+ - Sync up locales
```
Comment (BelleNottelling) [inline CHANGELOG.md:15]: Let's specifically note that FOSSBilling is 21% translated into 13 different languages and provide a link to https://translate.fossbilling.org/ for anyone who wants to help translate

**Review by BelleNottelling — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,19 @@
+## Version 0.2.6 (12-28-2022)
+
+### Security
+ - Introduce API wrapper for custom themes and modules to facilitate easier CSRF tokens (#612)
+
+## Bugfixes
+ - Fixed an issue with alias under Apache2 (#626)
+ - Add an additional check for CSFR when php://input is empty (#626)
```
Comment (BelleNottelling) [inline CHANGELOG.md:8]: I'd suggest this instead:
Better handle `php://input` being empty when checking the CSRF token.

**Review by BelleNottelling — APPROVED** [review]

**Pull request merged**